OloEngine/src/OloEngine/Networking/Prediction/NetworkMovementInput.cpp

```
@@ -0,0 +1,104 @@
+        // A NaN component would propagate into the transform and poison every
+        // downstream computation (culling, physics, the next snapshot) silently.
+        if (!std::isfinite(x) || !std::isfinite(y) || !std::isfinite(z))
+        {
+            return false;
+        }
+
+        out.Delta = { x, y, z };
+        return true;
+    }
+
+    InputApplyCallback MakeMovementApplyCallback(f32 maxStepDistance)
```

Cppcheck (2.21.0)

[style] The function 'MakeMovementApplyCallback' is never used.

(unusedFunction)